### PR TITLE
docs: add documentation framework related files

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+"""Configuration file for the Sphinx documentation builder."""
+import sys
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parents[2]))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "TensorBay"
```
Comment (linjiX) [inline docs/source/conf.py:26]: Do not use "TensorBay"

File: build_docs.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            ${{ hashFiles('requirements.txt') }}
+
+      - name: Set up python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          pip install sphinx furo
```
Comment (linjiX) [inline build_docs.yaml:42]: `pip install -r requirements_docs.txt`?

File: .readthedocs.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
```
Comment (linjiX) [inline .readthedocs.yaml:2]: 2022

**Review by linjiX — CHANGES_REQUESTED** [review]

File: .readthedocs.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+#
+
+---
```
Comment (linjiX) [inline .readthedocs.yaml:5]: Will it be better if the commit type is "docs"?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: docs/source/conf.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "furo"
+html_favicon = "images/favicon.svg"
```
Comment (linjiX) [inline docs/source/conf.py:75]: this file does not exist, will it make the build failed?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: docs/source/images/favicon.svg [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (linjiX) [inline docs/source/images/favicon.svg:1]: this icon seems useless

File: build_docs.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+          ruby-version: 2.6
+          bundler-cache: true
+
+      - name: Cache python ${{ matrix.python-version }}
+        uses: actions/cache@v2
+        with:
+          path: /opt/hostedtoolcache/Python/${{ matrix.python-version }}
+          key: >-
+            docs-${{ matrix.os }}-${{ matrix.python-version }}-
+            ${{ hashFiles('requirements.txt') }}
```
Comment (linjiX) [inline build_docs.yaml:33]: shall we add the hash of `requirements_docs.txt` here?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by AChenQ — APPROVED** [review]

**Review by linjiX — APPROVED** [review]

**Pull request merged**